src/kernelTypes/convCircKernel.jl

```diff
@@ -0,0 +1,92 @@
+export convCircKernel ,getConvCircKernel
+## For the functions nImgIn, nImgOut, nFeatIn, nFeatOut, nTheta, getOp, initTheta : see abstractConvKernel.jl
+## All convKernel types are assumed to have fields nImage and sK
+mutable struct convCircKernel{T} <: abstractConvKernel{T}
+    nImg :: Array{Int,1}
+    sK   :: Array{Int,1}
+    iK   :: Array{Int,1}
+end
+
+function nImgOut(this::convCircKernel)
+    return [this.nImg[1]; this.nImg[2]; this.sK[3]]
+ end
+
+function getConvCircKernel(TYPE::Type,nImg,sK)
```

Comment:
Just FYI outer constructors are usually named the same as the type, but this `getType` style is done throughout the package so we can just leave it for now